labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -19,7 +19,11 @@ object ListManipulationExercise02 {
    * of the two list
    */
   def sumOfTwo(l1: List[Int], l2: List[Int]): List[Int] = {
-    error("fix me")
+    (l1, l2) match {
+      case (Nil, ll2) => ll2
+      case (ll1, Nil) => ll1
+      case (ll1, ll2) => ll1.zip(ll2).map { case (a:Int,b:Int) => a+b }
+    }
```

Comment:
nice! we both used pattern matching but I did it in a bit different way:
---    (l1,l2) match {
      case (Nil,Nil) => List()
      case (x::xs,Nil) => (x::xs)
      case (Nil, y::ys) => (y::ys)
      case (x::xs,y::ys) => ((x+y)::sumOfTwo(xs,ys))
    }